Buildings/Controls/OBC/FDE/DOAS/DOAScontroller.mo

```diff
@@ -0,0 +1,494 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block DOAScontroller "DOAS controller built from DOAS blocks."
+
+  parameter Boolean vvUnit = true
+    "Set true when unit serves variable volume system."
+    annotation(Dialog(tab = "DOAS", group = "General"));
+
+  parameter Real damSet(
+    min = 0,
+    max = 1,
+    final unit = "1") = 0.9
+    "DDSP terminal damper percent open set point"
+    annotation(Dialog(tab = "DOAS", group = "General"));
+
+  parameter Real cctPIk = 0.0000001
+    "Cooling coil CCT PI gain value k."
+    annotation(Dialog(tab = "Cooling Coil", group = "PI Parameters"));
+
+  parameter Real cctPITi = 0.000025
+    "Cooling coil CCT PI time constant value Ti."
+    annotation(Dialog(tab = "Cooling Coil", group = "PI Parameters"));
+
+  parameter Real SAccPIk = 0.0000001
+    "Cooling coil SAT PI gain value k."
+    annotation(Dialog(tab = "Cooling Coil", group = "PI Parameters"));
+
+  parameter Real SAccPITi = 0.000025
+    "Cooling coil SAT PI time constant value Ti."
+    annotation(Dialog(tab = "Cooling Coil", group = "PI Parameters"));
+
+  parameter Real dehumSet(
```

Comment:
Please check and remove unused parameters

Reply:
Looks like none of the parameters are correctly assigned to the control submodules in this controller. Please assign required parameters, and remove unused ones